tests/test_v1_api.py

```diff
@@ -45,7 +45,11 @@ def test_comparison_report_api_post(self):
                 "11b3cbce-25a9-11e9-8457-c85b761454fa",
             ]
         }
-        response = self.client.post("api/drift/v1/comparison_report", data=data)
+        response = self.client.post(
+            "api/drift/v1/comparison_report",
+            data=data,
+            headers={"Content-Type": "application/json"},
```

Comment:
Is there a constant somewhere for the type?

Reply:
There is no constant for that and since it's only used in this one test I think it's fine to leave as is.